README.md

```diff
@@ -33,6 +33,8 @@ TODO:
 - [ ] Working bitcoin core rpc block-by-block example (see [#89](https://github.com/LLFourn/bdk_core_staging/pull/89))
 - [ ] Working bitcoin core rpc wallet sync example (see [#79](https://github.com/LLFourn/bdk_core_staging/pull/79))
 - [ ] Working compact block filters example using nakamoto (see #14).
```

Comment:
looks like I was wrong about being able to just use the #XX to link in README for a repo pr or issue.
```suggestion
- [ ] Working compact block filters example using nakamoto (see [#14](https://github.com/LLFourn/bdk_core_staging/pull/14)).
```